src/SAML2/Const.php

```diff
@@ -33,11 +33,21 @@ class SAML2_Const
     const BINDING_HTTP_ARTIFACT = 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Artifact';
 
     /**
+     * The URN for the PAOS binding.
+     */
+    const BINDING_PAOS = 'urn:oasis:names:tc:SAML:2.0:bindings:PAOS';
+
+    /**
      * The URN for the SOAP binding.
      */
     const BINDING_SOAP = 'urn:oasis:names:tc:SAML:2.0:bindings:SOAP';
 
     /**
+     * An internal binding that indicates that we should send a response by SOAP.
+     */
+    const BINDING_SOAP_RESPONSE = 'http://simplesamlphp.org/ns/bindings/SOAP-Response';
```

Comment:
This is the constant I was referring to before. We already have a `BINDING_SOAP` constant defining the proper namespace that identifies it. Bindings are defined for both requests and responses in the protocol (where applicable), so I don't see the need to define a specific constant carrying a custom namespace.